code/datums/mood_events/generic_positive_events.dm

```diff
@@ -166,6 +166,11 @@
 	mood_change = 4
 	timeout = 4 MINUTES
 
+/datum/mood_event/bottle_flip
+	description = "<span class='nicegreen'>The bottle landing like that was satisfying.</span>\n"
```

Comment:
```suggestion
	description = "[span_nicegreen("The bottle landing like that was satisfying.")]\n"
```

Reply:
this specific one; none of the other mood events are formatted like this-
I'll do it, just noting it